common/display/src/main/scala/uk/ac/wellcome/display/json/DisplayJsonUtil.scala

```diff
@@ -16,7 +18,7 @@ import uk.ac.wellcome.display.models.v2._
   *     render the complete value for disambiguation.
   *
   */
-trait DisplayJsonUtil extends AutoDerivation {
+trait DisplayJsonUtil extends AutoDerivation with TimeInstances {
```

Comment:
In general we've kept the “interesting” types out of the Display models.

They only contain primitive JSON types, like `String`, `Int` and `List`, and then their `apply()` method contains all the logic for serialising a Scala type (like LocalDate) into a String. It’s all kept together.

Could we push the LocalDate ~> String serialisation into `DisplayAccessCondition.apply()`? And be explicit about how we're doing it?

(I'm not completely opposed to changing it, but I'd like to at least discuss it.)

Reply:
:+1:  happy to keep to basic JSON types within the display if that is what we decided previously